### PR TITLE
Feature/take poster form input

### PR DESCRIPTION
**Pull request by ccollums**
**Is this a fix or a feature?**
A feature - when the user goes to make their own poster, they can input their values, and click show my poster to see their new poster. It also will save their values in the array variables. 
****What is the change?**
This changes the make your own poster page 
**What does it fix?**
N/A
**Where should the reviewer start?**
at the button query selector for show my poster button 
**Were there any issues that arose?**
we had all of our pieces of code written out but had a hard time piecing those together for a while. It came down to the order that we had things. 
**GOALS?****
The goal of this code was to have a functional form with user input.

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -182,6 +183,24 @@ function backToMainPage(){
   makePosterForm.classList.add('hidden');
   savedPostersPage.classList.add('hidden');
 }
+
+function captureFormValuesOnClick() {
```
Comment (Jayquintana) [inline src/main.js:187]: this one looks really good! I think we could combine the input lines together later when we refactor

File: src/main.js [hunk 2]
```diff
@@ -150,19 +159,11 @@ function getRandomIndex(array) {
  }
 
 
- function randomPosterButton() {
-   imageInput.innerText = getRandomIndex(images);
+ function randomPoster() {
```
Comment (Jayquintana) [inline src/main.js:162]: just noting to refactor this one later,

**Review by Jayquintana — APPROVED** [review]

**Comment by Jayquintana** [comment]
Looks Good! functions well, this one was a tough one and its overall really clean!

File: src/main.js (hunk 2 above)
Comment (tashiad) [inline src/main.js:162]: function names should generally start with an action verb, like `generateRandomPoster`

File: src/main.js (hunk 1 above)
Comment (tashiad) [inline src/main.js:187]: Don't forget to include `event` as a param

File: src/main.js [hunk 3]
```diff
@@ -182,6 +183,24 @@ function backToMainPage(){
   makePosterForm.classList.add('hidden');
```
Comment (tashiad) [inline src/main.js:183]: Think about how you can refactor all these show/hide functions using params!

**Review by tashiad — APPROVED** [review]
Looking great so far! One thing I'd recommend is to clear the form once the user has clicked the submit button. It's difficult to notice since currently when you hit submit it takes you to your new poster, but then if you navigate back to the form the values are still there.

**Pull request merged**